### PR TITLE
AST-153 - feat: Introduced new filter for updating Shopping Cart text in mobile flyout for Woo & EDD cart

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- As per user's support request we are introducing new filter which will help to update static "Shopping Cart" text in flyout 

### Screenshots
- https://a.cl.ly/jkue0OWm
- With using filter - https://a.cl.ly/qGuX8Gjm

### Types of changes
- Non-breaking change

### How has this been tested?
- Steps to replicate - https://a.cl.ly/jkue0O87
- `add_filter( 'astra_header_cart_flyout_shopping_cart_text', function() {
	return 'My Shop Flyout Cart';
} );`

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/builder/controllers/class-astra-builder-ui-controller.php [hunk 1]
```diff
@@ -281,10 +281,12 @@ public static function render_mobile_cart_flyout_markup() {
 							<?php echo self::fetch_svg_icon( 'close' ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
 					</button>
 					<div class="astra-cart-drawer-title">
-					Shopping Cart
+					<?php
+						echo apply_filters( 'ast_header_cart_flyout_shopping_cart_text', __( 'Shopping Cart', 'astra' ) ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```
Comment (premanshup) [inline inc/builder/controllers/class-astra-builder-ui-controller.php:285]: `ast_header_cart_flyout_shopping_cart_text` - @imnavanath This should be `astra_`

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/builder/controllers/class-astra-builder-ui-controller.php (hunk 1 above)
Comment (imnavanath) [reply inc/builder/controllers/class-astra-builder-ui-controller.php]: Updated with prefix - `astra`

**Review by premanshup — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

**Review by rohitp-bsf — APPROVED** [review]

File: changelog.txt [hunk 2]
```diff
@@ -1,6 +1,8 @@
 v3.5.0 (Unreleased)
 - New: Related Posts - Option to change "Related Posts" title.
 - New: Related Posts - Alignment option for "Related Posts" title.
+- Improvement: Added site title & tagline color options from Astra Pro to free theme.
+- Improvement: Builder - WooCommerce Cart & EDD Cart - Filter to change "Shopping Cart" text in responsive flyout.
```
Comment (premanshup) [inline changelog.txt:5]:
```suggestion
- Improvement: Builder - WooCommerce Cart & EDD Cart - Filter to change "Shopping Cart" text in responsive flyout. ( Need Doc )
```

File: changelog.txt [hunk 3]
```diff
@@ -1,6 +1,8 @@
 v3.5.0 (Unreleased)
 - New: Related Posts - Option to change "Related Posts" title.
 - New: Related Posts - Alignment option for "Related Posts" title.
+- Improvement: Added site title & tagline color options from Astra Pro to free theme.
```
Comment (premanshup) [inline changelog.txt:4]:
```suggestion
- Improvement: Added Site Title & Tagline color options from Astra Pro to free theme.
```

**Pull request merged**